public/docs/ts/latest/glossary.jade

```diff
@@ -431,29 +415,29 @@ a#H
 
 .l-main-section#L
 :marked
-  ## Lifecycle Hooks
+  ## Lifecycle hooks
 .l-sub-section
   :marked
-    [Directives](#directive) and [Components](#component) have a lifecycle
-    managed by Angular as it creates, updates and destroys them.
+    [Directives](#directive) and [components](#component) have a lifecycle
+    managed by Angular as it creates, updates, and destroys them.
 
-    Developers can tap into key moments in that lifecycle by implementing
+    You can tap into key moments in that lifecycle by implementing
     one or more of the "Lifecycle Hook" interfaces.
```

Comment:
"Lifecycle Hook" seems overcapitalized. I'm also not sure about the quotation marks. Maybe:

-> *lifecycle hook*